docs(rocprofiler-sdk): Update install instructions for 7.13 (#6034)
ROCm/rocm-systems: projects/rocprofiler-sdk/source/docs/install/build.rst

.. _installing-rocprofiler-sdk:

*********************************
Build ROCprofiler-SDK from source
*********************************

To build ROCprofiler-SDK as part of the ROCm Core SDK, see `TheRock build
instructions
<https://github.com/ROCm/TheRock/blob/main/docs/development/README.md>`__.
TheRock is the recommended way to build ROCm components from source.

> Alternatively, you can build ROCprofiler-SDK standalone using the following
> instructions.

Reviewer: ```suggestion
Alternatively, you can build ROCprofiler-SDK standalone using the instructions provided in the following sections.
```

The relevant fields are ``ID`` and the ``VERSION_ID``.

Build requirements
==================

To build on Linux, install the following dependencies:

.. tab-set::

   .. tab-item:: Debian-based distros

      .. code-block:: bash

          sudo apt install -y libdw-dev libsqlite3-dev

   .. tab-item:: RHEL-based distros

      .. code-block:: bash

          sudo dnf install elfutils elfutils-devel sqlite-devel clang-tools-extra gcc gcc-c++ cmake make openssl-devel
          python3 -m pip install --upgrade pip
          python3 -m pip install scikit-build

   .. tab-item:: SLES

      .. code-block:: bash

          sudo zypper install gcc12 gcc12-c++ cmake make python3-devel elfutils sqlite3-devel libelf-devel libdw-devel
          export CXX=/usr/bin/g++-12
          export CC=/usr/bin/gcc-12

      .. note::

>          The above ``export`` statements set the compiler environment variables only for the current terminal session. If you open a new terminal or log out, these variables will be unset. To make these settings permanent, add the following lines to your ``~/.bashrc`` file:

Reviewer: ```suggestion
         The preceding ``export`` statements set the compiler environment variables only for the current terminal session. Opening a new terminal or logging out unsets these variables. To make these settings permanent, add the following lines to your ``~/.bashrc`` file:
```
